lib/docker.ts

```diff
@@ -135,14 +139,17 @@ class Docker {
    * Find files on a docker image according to a given list of glob expressions.
    */
   public async findGlobs(
-    globs: string[],
+    globs: any,
```

Comment:
why?

Reply:
globs is an object now of:
manifestGlobs && binariesGlobs

I'll make it into a type